src/shibuya.js

```diff
@@ -28,24 +28,24 @@ const main = async () => {
     const proxyOnShibuya = shibuyaHelper.getProxyAccount(TURING_PARA_ID, shibuyaAddress);
 
     console.log(`\na) Add a proxy of Turing (paraId:${TURING_PARA_ID}) for Alice on Shibuya ...\n Proxy address: ${proxyOnShibuya}\n`);
-    await sendExtrinsic(shibuyaHelper.api, shibuyaHelper.api.tx.proxy.addProxy(proxyOnShibuya, 'Any', 0), keyPair);
```

Comment:
Astar has created two new types for their proxy. "Any" will not work on Shiden, so we will need to update the addProxy() call.

The two types are,
```
pub enum ProxyType {
    CancelProxy,
    DappsStaking,
}
```

Reference:
https://github.com/AstarNetwork/Astar/blob/523c06798a08189a3ea20f790b83cd4ae602c579/runtime/shiden/src/lib.rs#L834

Reply:
Since we need to test both schedule and cancel with the two types, instead of `{ Fixed: { executionTimes: [0] } }` please use `Recurring` to trigger the task every 3 minutes. Wait for 3 minutes for the first trigger, and then call cancel task from Shiden, and verify that the second trigger does not happen at the 6th minute.

Reply:
Instead of "Any", we should examine and use "DappsStaking".

Reply:
Fixed